framework/moose.mk

```diff
@@ -31,12 +31,17 @@ hit_deps      := $(patsubst %.cc, %.$(obj-suffix).d, $(hit_srcfiles))
 # hit python bindings
 #
 pyhit_srcfiles  := $(hit_DIR)/hit.cpp $(hit_DIR)/lex.cc $(hit_DIR)/parse.cc
-pyhit_LIB       := $(hit_DIR)/hit.so
+pyhit_LIB       := $(FRAMEWORK_DIR)/../python/hit.so
 
 $(pyhit_LIB): $(pyhit_srcfiles)
 	@echo "Linking Library "$@"..."
 	@bash -c '(cd "$(hit_DIR)" && $(libmesh_CXX) -std=c++11 -w -fPIC -lstdc++ -shared -L`python-config --prefix`/lib `python-config --includes` `python-config --ldflags` $^ -o $@)'
-	@cp $@ $(FRAMEWORK_DIR)/../python/
+
+# This target can be manually called by users to (force re)build the hit python bindings.
+hit: | rmhit $(pyhit_LIB)
```

Comment:
What is the purpose of `|` here?

Reply:
order-only dependencies (special makefile syntax)